app/models/invoice/data_presenter/line_item.rb

```diff
@@ -12,8 +12,20 @@ class LineItem < Invoice::DataPresenter::Base
 
       delegate :name_to_display, :options_text, to: :variant
 
-      def display_amount_with_adjustments
-        Spree::Money.new((price_with_adjustments * quantity), currency:)
+      def amount_with_adjustments_without_taxes
+        (price_with_adjustments * quantity) - included_tax
+      end
+
+      def amount_with_adjustments_and_with_taxes
+        ( price_with_adjustments * quantity) + added_tax
+      end
```

Comment:
These one-line methods aren't used anywhere else, so could be combined with the corresponding `display_` methods below 👇